balancer-js/src/modules/pools/pool-types/concerns/stablePhantom/priceImpact.spec.ts

```diff
@@ -0,0 +1,76 @@
+import dotenv from 'dotenv';
+import { expect } from 'chai';
+import { StablePhantomPriceImpact } from '@/modules/pools/pool-types/concerns/stablePhantom/priceImpact.concern';
+
+import pools_14717479 from '@/test/lib/pools_14717479.json';
+import { StaticPoolRepository } from '@/modules/data';
+import { PoolsProvider } from '@/modules/pools/provider';
+import { PoolModel, Pool } from '@/types';
+import { Network } from '@/.';
+import { setupPool } from '@/test/lib/utils';
+
+dotenv.config();
+const rpcUrl = 'http://127.0.0.1:8545';
+const priceImpactCalc = new StablePhantomPriceImpact();
+const bbaUSDPoolId =
+  '0x7b50775383d3d6f0215a8f290f2c9e2eebbeceb20000000000000000000000fe';
+
+describe('phantomStable pool price impact', () => {
+  let pool: PoolModel | undefined;
```

Comment:
The updated version of `setupPool` already returns a `PoolModel` instead of `PoolModel | undefined`, so you should be able to refactor this.